app/controllers/portal/client_logins_controller.rb

```diff
@@ -20,7 +20,8 @@ def create
           phone_number: @form.sms_phone_number,
           visitor_id: visitor_id,
           locale: I18n.locale,
-          service_type: service_type
+          service_type: service_type,
+          state_code: state_code
```

Comment:
intake_logins_controller inherits from this so when dealing with gyr clients it'll have a nil state code and when statefile it will pass in a state code. this is used to generate the url that goes in the message